src/main/java/Matchers/Implementations/IngredientMatcher.java

```diff
@@ -40,4 +41,30 @@ public double floatMatch(Recipe recipe) {
                 counter++;
         return (double) counter/recipeItems.size();
     }
+
+    /**
+     * Take in a recipe storage and return a list of the top 10 (less if there isn't 10) most similar recipes
+     * @param recipes
+     */
+    public List<Recipe> return10RecipesMatched(RecipeStorage recipes) {
+        List<Recipe> allRecipes = new ArrayList<>();
```

Comment:
If you do something like this, chances are very large you should have just done "top x" and not hard code it. Also, naming functions with things like "return" is a little odd. Just doesn't really flow when talking / describing it.